### ISSUE TITLE
Guide to Image Target Detection

### ISSUE THREAD

**Issue by lisa92203** [8thwall/xr-unity#53]
I saw **XRImageDetectionController.cs** and **XRImageDetectionTargetController.cs** in the scripts folder, but they are not mentioned in README. Is there any guide to image-target detection? Thank you!

**Comment by dattanchu** [comment]
Unfortunately, there is no guide at the moment. While we are working on a guide, let me know if these crude instructions help.

1) You need to put XRImageDetectionController onto a GameObject
2) You need your image targets as Texture in your project. Mark them "Read/Write Enabled".  See the documentation of its programmatic usage here https://docs.8thwall.com/#xrcontrollersetdetectionimages
3) You can then put the XRImageDetectionTargetController onto a target object. Reference the XRImageDetectionController on step 1. Choose the target texture that this target should respond to, then register the callback. There is a MoveToTarget callback that XRImageDetectionTargetController already exposes. This callback will move this object to wherever the image target is detected.

In other words, let's say you have an image of the number 8 in the scene. You can attach the XRImageDetectionTargetController to your bunny game object with MoveToTarget callback. When the number 8 is detected, your bunny will be moved to the number 8.

**Comment by garayco** [comment]
I don't understand. Can I put a Image Target to show a 3D Object ?

**Comment by dattanchu** [comment]
@garayco Yes you can. You simply attach the XRImageDetectionTargetController to your 3D Object. The long info is to follow the 3 steps I mentioned above.

**Comment by garayco** [comment]
![captura](https://user-images.githubusercontent.com/22529692/43045274-3274f778-8d7b-11e8-96b3-ff0eb94da5b1.JPG)

something like this ? 

I created a GameObject that have the XRImageDetectionController script.

**Comment by garayco** [comment]
![image](https://user-images.githubusercontent.com/22529692/43049036-fcb98482-8db6-11e8-8fba-35747402928b.png)

I have to set this?

and 

What should I put here?

![image](https://user-images.githubusercontent.com/22529692/43049044-1d33d55a-8db7-11e8-804d-2e23b6806e25.png)

I apologize for the inconvenience, but I'm new with some things.

**Comment by dattanchu** [comment]
XRImageDetectionController Tex should match the target texture in XRImageDetectionTargetController.
There is only one object in that list which has MoveToTarget which is XRImageDetectionTargetController.

**Comment by salvator12** [comment]
I have follow the guide, but it's not works

**Comment by atomarch** [comment]
Please see comments here: https://github.com/8thwall/xr-unity/issues/78#issuecomment-449915109
